src/components/characters/CharacterList.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Character from './Character';
+import { useCharacters } from '../state/CharacterProvider';
+
+const CharacterList = () => {
+  const characters = useCharacters();
```

Comment:
yep, you don't need to pass a prop, just useCharacter and you've got your characters.